### PR TITLE
Added initial version of about view.

### PR DESCRIPTION
**Pull request by GijsWeterings**

### CODE REVIEW

File: app/views/about.html [hunk 1]
```diff
@@ -1 +1,43 @@
-<p>This is the about view.</p>
+<div class="about">
+    <h1>Welcome to MoodCat.me</h1>
+    <p>
+        Hi there!
+        Welcome to MoodCat.me, the music platform of the future, available today.
+        Join us in the adventure of finding and enjoying music geared to you!
+        We try to understand you
```
Comment (JeremybellEU) [inline app/views/about.html]: This sentence sounds a bit weird, maybe it can just be omitted?

File: app/views/about.html [hunk 2]
```diff
@@ -1 +1,43 @@
...
+    </p>
+    <h2>How does it work?</h2>
+    <p>
+        Go to the homepage, and select what mood(s) you're in.
+        Once you click next, we recommend you a couple of rooms to check out.
```
Comment (JeremybellEU) [inline app/views/about.html]: 'we provide a list of possible suitable rooms you can join.'

File: app/views/about.html [hunk 3]
```diff
@@ -1 +1,43 @@
...
+        Pick one you think you like, and you will join the room.
```
Comment (JeremybellEU) [inline app/views/about.html]: 'Pick the one you like to join the room.' it sounds more direct and clear for the user.

File: app/views/about.html [hunk 4]
```diff
@@ -1 +1,43 @@
...
+        We'll start playing music for you, and you can chat with people in the same mood as you are!
```
Comment (JeremybellEU) [inline app/views/about.html]: 'Immediately music will start playing', we don't do anything, the system does. Also the comma before the and could be omitted.

'Immediately music will start playing and you can interact with people in the same mood.'

File: app/views/about.html [hunk 5]
```diff
@@ -1 +1,43 @@
...
+    </p>
+    <blockquote>
+        This song doesn't fit the room, why am I hearing this?
```
Comment (JeremybellEU) [inline app/views/about.html]: 'The song that is currently playing doesn't fit'

File: app/views/about.html [hunk 6]
```diff
@@ -1 +1,43 @@
...
+    </blockquote>
+    <p>
+        We're constantly trying to improve the system. You can give us feedback to help us.
+        If you think a song fits the current mood, click the <i class="glyphicon glyphicon-thumbs-up"></i> button in the room.
+        This way, we know whe made the right recommendation. Similarly, if the song does not fit the room, click the <i class="glyphicon glyphicon-thumbs-down"></i> button.
```
Comment (JeremybellEU) [inline app/views/about.html]: 'This will tell the system it provided the correct recommendation'

File: app/views/about.html [hunk 7]
```diff
@@ -1 +1,43 @@
...
+        We'll try to modify the system based on your feedback. If you want to help us even better, you can use the popup to modify the rating of the song to your vision.
```
Comment (JeremybellEU) [inline app/views/about.html]: 'We'll modify the system', we don't try, we do.

File: app/views/about.html [hunk 8]
```diff
@@ -1 +1,43 @@
...
+    </p>
+    <h3>We need your help!</h3>
```
Comment (JeremybellEU) [inline app/views/about.html]: 'Other ways to improve the system' This subject is very similar to the previous paragraph.

File: app/views/about.html [hunk 9]
```diff
@@ -1 +1,43 @@
...
+    <p>
+        Want to make MoodCat a better place?
+        You can help us improving Moodcat in a couple of ways.
+        All users can participate in our <a href="/rate-the-song">rating game</a>.
+        Listen to 30 seconds of music, and give your feedback on the song.
+        For every 5 songs you rate, you earn <strong>30</strong> points!
+    </p>
+    <p>
+        But there are even more ways to earn points. If you find a bug in MoodCat,
```
Comment (JeremybellEU) [inline app/views/about.html]: But can be omitted, it is not a contraposition, but a continuation of the previous point.

File: app/views/about.html [hunk 10]
```diff
@@ -1 +1,43 @@
...
+        <a href="https://github.com/MoodCat/MoodCat.me/issues/new">file a Github issue!</a>.
+        Correct and useful bug reports can earn you up to <strong>100</strong> points!
+    </p>
+    <p>
+        Have you thought of a cool new feature, and do you want to share it with everybody?
```
Comment (JeremybellEU) [inline app/views/about.html]: 'Do you have a cool feature in mind?'

File: app/views/about.html (hunk 8 above)
Comment (GijsWeterings) [inline app/views/about.html]: true, but "We need your help!" is a call to action. Other ways... is just TL;DR

**Comment by JeremybellEU** [comment]
Yes looks good! Merging beep boop.

**Pull request merged**